src/app/pages/datasets/modules/permissions/stores/dataset-acl-editor.store.ts

```diff
@@ -66,6 +66,15 @@ export class DatasetAclEditorStore extends ComponentStore<DatasetAclEditorState>
           this.ws.call('filesystem.stat', [mountpoint]),
         ]).pipe(
           tap(([acl, stat]) => {
+            acl.acl.forEach((ace, aceIndex) => {
+              if ((!ace.who && [NfsAclTag.User, PosixAclTag.User].includes(ace.tag))
```

Comment:
Let's make these conditionals into named variables that explain what this condition means. such as 
`const aceIsUser = !ace.who && [NfsAclTag.User, PosixAclTag.User].includes(ace.tag);
...
if(aceIsUser && ...)`

Reply:
I have added two constants` unknownUser`, `unknownGroup`. When the store is loaded, a check is made to see if the deleted user or group exists (`!ace.who`).